src/schema/palette.ts

```diff
@@ -22,7 +22,14 @@ export type ContinuousScalePalette = PaletteBaseInfo & {
   colors: ContinuousColor[];
 };
 
+export type MatrixPalette = PaletteBaseInfo & {
+  type: "matrix";
+  colors: [Color[], Color[]];
```

Comment:
this prop could be more semantical?
if a matrix has only two dims, could be colors.x , colors.y ?